### DISCUSSION TITLE
Is there a next version roadmap published?

### DISCUSSION THREAD

**Discussion by alejandrofm** [duckdb/duckdb#8673] [General]
I like to look at the features for next releases, sometimes just for the hype, sometimes because maybe some issue I'm facing has already been worked on.
I saw that the project used milestones up to version 0.7 and the only data I could find was the next release date.
Where I can see the issues/new features/etc WIP or DONE for the next release?
Thanks!

**Comment by szarnyasg** [comment]
Hi @alejandrofm, we do not have a public roadmap at the moment. You can see the planned release dates (without features) at https://duckdb.org/dev/release-dates.html.

**Reply by alejandrofm** [reply]
Thanks @szarnyasg it would be nice, if not the roadmap, maybe the changelog so far, or main solved issues until now.
this will also avoid reporting solved issues and will give some idea on things to look forward.
Also if the "want to do for next release" issues got shared, the community can prioritize where the help is needed.
